source/agora/node/Ledger.d

```diff
@@ -1160,6 +1162,49 @@ public class ValidatingLedger : Ledger
         assert(data.tx_set.length - pre_cb_len <= 1);
     }
 
+    /***************************************************************************
+
+        Calculate the transaction fee and adjust the fee based on the
+        transaction's size measured in bytes.
+
+        The bigger the transaction size is, the smaller the the fee becomes.
```

Comment:
Is this right? Shouldn't the fee be higher for a bigger size?

Reply:
> Is this right? Shouldn't the fee be higher for a bigger size?

oh thx, I made a mistake in the doc, I should have said:
The bigger the transaction size is, the smaller the **adjusted** fee becomes. So if we have 2 txs, each pays 10 coin fee, but tx1 has double the size of tx2, then tx1 has 2 times smaller **adjusted** fee.